### PR TITLE
fix iscmagic scope

### PR DESCRIPTION
**Pull request by luke-thorne**
# Description of change

Update scope of `iscmagic` npm package so we deploy to `@iota/iscmagic`.

## Links to any relevant issues

#2470

### CODE REVIEW

File: packages/vm/core/evm/iscmagic/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# @iota/iscmagic
+
+[EVM and ISC are inherently very different platforms](https://wiki.iota.org/shimmer/smart-contracts/guide/evm/compatibility/). Some EVM-specific actions (e.g., manipulating Ethereum tokens) are disabled, and EVM contracts can access ISC-specific functionality through the ISC Magic Contract.
```
Comment (jorgemmsilva) [inline packages/vm/core/evm/iscmagic/README.md]: I'm not sure what is meant by:
> Some EVM-specific actions (e.g., manipulating Ethereum tokens) are disabled

**Review by jorgemmsilva — APPROVED** [review]

File: packages/vm/core/evm/iscmagic/README.md (hunk 1 above)
Comment (luke-thorne) [reply packages/vm/core/evm/iscmagic/README.md]: I copied it from the wiki. That is probably from an old implementation and needs to be updated. @dessaya? 
https://wiki.iota.org/shimmer/smart-contracts/guide/evm/magic/

Comment (dessaya) [reply packages/vm/core/evm/iscmagic/README.md]: Yes, that text is somewhat outdated. Maybe we can even remove this paragraph.

**Review by dessaya — APPROVED** [review]

File: packages/vm/core/evm/iscmagic/README.md (hunk 1 above)
Comment (luke-thorne) [inline packages/vm/core/evm/iscmagic/README.md]:
```suggestion

```

remove outdated paragraph

Comment (luke-thorne) [reply packages/vm/core/evm/iscmagic/README.md]: removed it

**Pull request merged**